apps/demo-game/package.json

```
@@ -26,8 +26,9 @@
     "formik": "2.4.6",
     "graphql": "16.9.0",
     "graphql-scalars": "1.23.0",
-    "graphql-sse": "2.5.3",
+    "graphql-sse": "2.5.4",
     "graphql-yoga": "4.0.4",
+    "@graphql-yoga/plugin-graphql-sse": "^3.13.5",
```

```suggestion
    "@graphql-yoga/plugin-graphql-sse": "3.13.5",
```

Pin and regenerate lockfile

---

And reevaluate if we need it, would not be necessary if using the natively supported distinct mode.